src/bundle/Core/Command/ExpireUserPasswordsCommand.php

```diff
@@ -0,0 +1,383 @@
+     * @param array<string> $userContentTypeIdentifiers
+     */
+    private function summarizeSearchCriteria(
+        OutputInterface $output,
+        $userIds,
+        $userGroupIds,
+        $userContentTypeIdentifiers
+    ): void {
+        $output->writeln('<info>Criteria used to find users:</info>');
+
+        if (!empty($userIds)) {
+            $output->writeln(
+                sprintf("<info>\tUser ID: %s</info>", implode(', ', $userIds))
+            );
+        }
+
+        if (!empty($userGroupIds)) {
+            $output->writeln(
+                sprintf("<info>\tUser Group ID: %s</info>", implode(', ', $userGroupIds))
+            );
+        }
+
+        if (!empty($userContentTypeIdentifiers)) {
+            $output->writeln(
+                sprintf(
+                    "<info>\tUser Content Type Identifier: %s</info>",
+                    implode(', ', $userContentTypeIdentifiers)
+                )
+            );
+        }
+    }
+
+    /**
+     * @param array<string, mixed> $fieldSettings
+     * @param array<string, mixed> $validatorConfiguration
+     */
+    private function updateContentType(
+        ContentType $contentType,
+        array $fieldSettings,
+        array $validatorConfiguration,
+        FieldDefinition $fieldDefinition
+    ): void {
+        $this->repository->sudo(
+            static function (
+                Repository $repository
+            ) use (
+                $contentType,
+                $fieldSettings,
+                $validatorConfiguration,
+                $fieldDefinition
+            ): void {
+                $contentTypeDraft = $repository
+                    ->getContentTypeService()
+                    ->createContentTypeDraft($contentType);
+                $fieldDefinitionUpdateStruct = $repository
+                    ->getContentTypeService()
+                    ->newFieldDefinitionUpdateStruct();
+
+                $fieldDefinitionUpdateStruct->fieldSettings = $fieldSettings;
+                $fieldDefinitionUpdateStruct->validatorConfiguration = $validatorConfiguration;
+
+                $repository
+                    ->getContentTypeService()
+                    ->updateFieldDefinition(
+                        $contentTypeDraft,
+                        $fieldDefinition,
+                        $fieldDefinitionUpdateStruct
+                    );
+
+                $repository
+                    ->getContentTypeService()
+                    ->publishContentTypeDraft($contentTypeDraft);
+            }
+        );
+    }
+
+    /**
+     * @param array<\eZ\Publish\API\Repository\Values\Content\Field> $fields
+     */
+    private function findUserField(array $fields): ?Field
+    {
+        $userField = null;
+
+        foreach ($fields as $field) {
+            if ($field->fieldTypeIdentifier !== 'ezuser') {
+                continue;
+            }
+            $userField = $field;
+        }
+
+        return $userField;
+    }
```

Comment:
`ezuser` FT is is singular so you can simplify this method to:

```suggestion
    /**
     * @param array<\eZ\Publish\API\Repository\Values\Content\Field> $fields
     */
    private function findUserField(array $fields): ?Field
    {
        foreach ($fields as $field) {
            if ($field->fieldTypeIdentifier === 'ezuser') {
                return $field;
            }
        }

        return null;
    }
```